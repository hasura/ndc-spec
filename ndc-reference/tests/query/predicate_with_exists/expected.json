[
  {
    "rows": [
      {
        "first_name": {
          "value": "John"
        },
        "articles": {
          "rows": {
            "rows": [
              {
                "title": {
                  "value": "Why Functional Programming Matters"
                },
                "id": {
                  "value": 2
                }
              },
              {
                "id": {
                  "value": 3
                },
                "title": {
                  "value": "The Design And Implementation Of Programming Languages"
                }
              }
            ]
          }
        },
        "last_name": {
          "value": "Hughes"
        }
      }
    ]
  }
]