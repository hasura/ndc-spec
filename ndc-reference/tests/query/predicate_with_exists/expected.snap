---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/predicate_with_exists/request.json
---
[
  {
    "rows": [
      {
        "first_name": "John",
        "last_name": "Hughes",
        "articles": {
          "rows": [
            {
              "id": 2,
              "title": "Why Functional Programming Matters"
            },
            {
              "id": 3,
              "title": "The Design And Implementation Of Programming Languages"
            }
          ]
        }
      }
    ]
  }
]
